style.css

```diff
@@ -205,6 +213,61 @@ body {
   font-weight: 500;
   margin-bottom: 40px;
   border: none;
+  transition: all 1s linear;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+}
+
+.contact h2 {
+  color: #9390ff;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  width: 75%;
+}
+
+form input {
+  margin: 10px 0;
+  width: 100%;
+  height: 60px;
```

Comment:
- [ ] I can see in the design that the height of these input elements is `48px`. Please change the height for the name and email input elements to `48px`.